src/Lab.3.Bouncer/Lab.3.Bouncer.cs

```diff
@@ -0,0 +1,38 @@
+namespace PEURandom;
+using System.Management.Automation;
+using LoremNET;
+
+[Cmdlet(VerbsCommon.Get, $"{Consts.ModulePrefix}Age")]
+public class GetPEURandomAgeCmdlet : PSCmdlet
+{
+    [Parameter(ValueFromPipeline = true, Mandatory = true, Position = 0)]
+    public string name { get; set; } = "";
+    protected override void ProcessRecord()
+    {
+        Person person = new Person(name, Lorem.DateTime(1900));
+        WriteObject(person);
+    }
+}
+
+[Cmdlet("Assert", $"{Consts.ModulePrefix}Age")]

```

Comment:
Just like with VerbsCommon.Get on line 5, you can use VerbsLifecycle.Assert to reference the string from the approved verbs. It's the same result though, just a small tip :)